### PR TITLE
Add missing shell setup in README

### PR DESCRIPTION
**Pull request by pcanal**
Sorry for the omission in the prevision PR

### CODE REVIEW

**Comment by esseivaju** [comment]
the links to the installation documentation are also broken, maybe we can update it at the same time?

**Comment by pcanal** [comment]
Which ones?  Some were fixed in [the previous PR ](https://github.com/celeritas-project/celeritas/pull/1532)

**Comment by esseivaju** [comment]
[L73](https://github.com/celeritas-project/celeritas/blob/d1d17655eec50fe4031018e93a407147d309e02d/README.md#L73) and [L132](https://github.com/celeritas-project/celeritas/blob/d1d17655eec50fe4031018e93a407147d309e02d/README.md#L132) are pointing to the same dead link

File: README.md [hunk 1]
```diff
@@ -130,7 +130,7 @@ The full set of configurations is viewable on CI platform [GitHub Actions][gha])
 Compatibility fixes that do not cause newer versions to fail are welcome.
 
 [spack]: https://github.com/spack/spack
-[install]: https://celeritas-project.github.io/celeritas/user/main/installation.html
+[install]: https://celeritas-project.github.io/celeritas/user/introduction/installation.html
```
Comment (esseivaju) [inline README.md:133]: I think the other install link at line 73 needs to be updated as well (or removed)

Comment (pcanal) [reply README.md:133]: A priorio they are the same references?

Comment (pcanal) [reply README.md:133]: annoyingly they are not ... pushed another update.

**Review by esseivaju — APPROVED** [review]

**Pull request merged**